scripts/eval/eval.py

```diff
@@ -170,15 +159,14 @@ def evaluate_model(model_cfg: DictConfig, dist_timeout: Union[float, int],
         torch.cuda.synchronize()
     b = time.time()
     print(f'Ran {model_cfg.model_name} eval in: {b-a} seconds')
-    return (in_memory_logger, logger_keys, model_gauntlet_callback,
-            model_gauntlet, model_gauntlet_df)
+    return (trainer, logger_keys, eval_gauntlet_callback, eval_gauntlet_df)
 
 
 def main(cfg: DictConfig):
     om.resolve(cfg)
     model_configs: ListConfig = pop_config(cfg, 'models', must_exist=True)
-    model_gauntlet_config: Optional[Union[str, DictConfig]] = pop_config(
-        cfg, 'model_gauntlet', must_exist=False, default_value=None)
+    eval_gauntlet_config: Optional[Union[str, DictConfig]] = pop_config(
```

Comment:
same comment as train script. need to not break existing yamls, so look for both old key and new key.